.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "extends": [
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
I guess this does not interfere with the config that was added to the package by create-react-app?
It would also be nice to make sure that integration still works and linting is not only possible with the linting task but rather still done during development and building the app

Reply:
The `start` and `build` scripts use eslint with the new config, but it looks like files that are not part of the bundle (e.g. tests) are ignored. That's why I added `lint`.

Reply:
Is see, perfect